### DISCUSSION TITLE
[SOLVED]GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"

### DISCUSSION THREAD

**Discussion by rodrimarchese** [community/community#13770] [Copilot Conversations] [answered]
I'm now autorized to use copilot but I intall it and this happends:

```
Invalid copilot token: missing token: 404 
GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"
```

I'va already logged out and in, my github account and reinstalled copilot. 

Any ideas? 

<img width="733" alt="image" src="https://user-images.githubusercontent.com/32769860/160633038-e6810f87-c4bc-4c6c-868e-b99a27d41f3a.png">

<img width="1232" alt="image" src="https://user-images.githubusercontent.com/32769860/160633287-926fa625-bffa-4da9-b108-420d7e18b11e.png">

**Comment by altacountbabi** [comment]
so how did u solve it?

**Comment by bl-hoang** [comment, marked as answer]
Same issue here when I re-enabling the extension after disabling.

**Reply by davecheney** [reply]
@[bl-hoang](https://github.com/bl-hoang) you are not on the waitlist, please register https://github.com/features/copilot/signup and wait for an invitation.
